app/src/main/java/org/mozilla/tv/firefox/fxa/FxaRepo.kt

```diff
@@ -142,46 +136,6 @@ class FxaRepo(
         }
     }
 
-    @VisibleForTesting(otherwise = PRIVATE)
-    inner class FirefoxDeviceEventsObserver : DeviceEventsObserver {
-        override fun onEvents(events: List<DeviceEvent>) {
-            logger.debug("received device events: $events")
-            events.forEach {
-                when (it) {
-                    is DeviceEvent.TabReceived -> onReceiveTab(it)
-                }
-            }
-        }
-
-        private fun onReceiveTab(event: DeviceEvent.TabReceived) {
-            if (event.entries.isEmpty()) {
-                sentryIntegration.captureAndLogError(logger, ReceiveTabException("Received push event with no entries"))
-                return
-            }
-
-            val deviceMetadata = event.from?.let {
-                ReceivedTabs.DeviceMetadata(
-                    displayName = it.displayName,
-                    deviceType = it.deviceType
-                )
-            }
-
-            val receivedTab = ReceivedTabs(
-                urls = event.entries
-                    .map { it.url }
-                    .filter { it.isNotBlank() }, // blank URLs may not be possible but we do this just to be safe.
-                sendingDevice = deviceMetadata
-            )
-
-            if (receivedTab.urls.isEmpty()) {
-                sentryIntegration.captureAndLogError(logger, ReceiveTabException("Received push event with only blank URLs"))
-                return
-            }
-
-            _receivedTabs.onNext(receivedTab)
-        }
-    }
-
     companion object {
         private const val CLIENT_ID = "85da77264642d6a1"
```

Comment:
This isn't part of the PR, but what is this id? Can we link to where it comes from?